### ISSUE TITLE
TurboTable - Sort does not apply to filteredValue (search + sort bug)

### ISSUE THREAD

**Issue by bias-keenly** [primefaces/primeng#4875]
### There is no guarantee in receiving a response in GitHub Issue Tracker, If you'd like to secure our response, you may consider *PrimeNG PRO Support* where support is provided within 4 business hours

**I'm submitting a ...**  (check one with "x")
```
[X] bug report => Search github for a similar issue or PR before submitting
[X] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```
**Current behavior**
My table has both global search and sorting enabled.  When filtering the table using search the filteredValue is rendered.  When sorting with a filtered dataset the sort does not apply to filteredValue, and no sort action takes place.  

To put it another way, when searching you cannot sort.

**Expected behavior**
Ability to sort when filtering.  Apply sort to both value and filteredValue.

**Minimal reproduction of the problem with instructions**
Create table with both search and sort.  Search to filter table, then sort.  Sorting does not apply.

**What is the motivation / use case for changing the behavior?**

**Please tell us about your environment:**

* **Angular version:** 5.2.0

* **PrimeNG version:** 5.2.0-rc1

* **Browser:** [all]

**Comment by bias-keenly** [comment]
Re-applying the filterGlobal in onSort() is a workaround if anyone else runs into this.

**Comment by cagataycivici** [comment]
4910
